src/components/OSCALControl.js

```diff
@@ -61,6 +62,12 @@ export default function OSCALControl(props) {
     );
   }
 
+  // Retrieve implemented requirements set-parameters
+  const setParams = getImplReqSetParameters(
```

Comment:
Sorry, I meant the function and the execution of that function could be moved to `OSCALControlProse`.

There shouldn't be any need to evaluate here and pass the results as `implReqSetParameters ` all the way down to `OSCALControlProse` where it's actually used, I think we should be able to just call from within `OSCALControlProse`.